client/chat/channel-info-card.tsx

```diff
@@ -46,12 +54,26 @@ const ChannelCardBadge = styled.div`
   border-radius: 9999px;
 `
 
-const ChannelName = styled.div`
+const ChannelNameContainer = styled.div`
+  position: relative;
+`
+
+const ChannelName = styled.div<{ $hasOverflowMenu: boolean }>`
   ${headline6};
+  ${singleLine};
   margin-top: 4px;
+  margin-right: ${props => (props.$hasOverflowMenu ? '48px' : '0px')};
```

Comment:
Yeah, the only reason why I added this is because the overflow menu button is absolutely positioned. Maybe I can just position it relatively to align it better without taking it out completely out of the flex layout. Or group the channel name + user count together in a flex-column which is in a flex-row together with the overflow button.

Reply:
That's what I'm saying, you shouldn't absolutely position the overflow menu button. You are reimplementing a layout algorithm in a bad way by doing what you're doing here

Reply:
If the channel name is where League names are, I don't quite see why this padding is necessary though (the overflow menu should probably be at the top right of the card, and thus these elements are not adjacent ever). If that is the case, then absolute positioning is fine (but you don't need what you're doing here at all). I assume you have instead put it by the title, where absolute positioning makes zero sense. But I think I would prefer you just move it to the top right. Either way, this kind of prop on this style is unnecessary.